contribs/gmf/src/mobile/measure/lengthComponent.js

```diff
@@ -13,7 +13,9 @@ goog.require('ol.style.RegularShape');
 goog.require('ol.style.Stroke');
 goog.require('ol.style.Style');
 
-gmf.mobile.measure.lengthComponent = angular.module('', []);
+gmf.mobile.measure.lengthComponent = angular.module('', [
```

Comment:
? no name ? I I pretty sure we need one !

Reply:
Relay missing ...